modules/core/src/v2/coins/algo.ts

```diff
@@ -121,7 +121,7 @@ export class Algo extends BaseCoin {
    */
   valuelessTransferAllowed(): boolean {
     // TODO: this sounds like its true with the staking txes - confirm before launch
-    return false;
+    return true;
```

Comment:
The valueless transfer is not always allowed, hence returning it as true is not correct.

It is allowed for certain transaction types (enableToken) and that is what we shall implement. Enhance this method to accept optional transaction type param and based on it return `true` or `false`